opds_import.py

```diff
@@ -532,6 +532,29 @@ class OPDSImporter(object):
             "label": _("Password"),
             "description": _("If HTTP Basic authentication is required to access the OPDS feed (it usually isn't), enter the password here."),
         },
+        {
+            "key": ExternalIntegration.CUSTOM_ACCEPT_HEADER,
+            "label": _("Custom accept header"),
+            "required": False,
+            "description": _("Some servers expect an accept header to decide which file to send. You can use */* if the server doesn't expect anything. The default values if left blank is: 'application/atom+xml;profile=opds-catalog;kind=acquisition, application/atom+xml;q=0.9, application/xml;q=0.8, */*;q=0.1'"),
```

Comment:
Rather than mentioning this complex default in the description, make it the default value as part of the configuration. This will let you get rid of the `_get_accept_header` method altogether.

Reply:
I totally agree about changing the description to get smaller one and use a default value.
Without running a script to populate the db with the values for the existing collections we cannot get rid of the `_get_accept_headers` method, so I’ll let the method existing to keep the backwards compatibility.

Reply:
You could write a database migration script for this and add it to this branch, but it shouldn't be necessary: the default value will be used if a collection has no ConfigurationSetting object for this setting.

Try it out, but setting a default and removing the code should make everything work as it did before.

Reply:
I ran 2 tests: without changing the code and removing the `_get_accept_header method`, both with the same database config.

Without changing the code, I got this accept in request header:
`'Accept': 'application/atom+xml;profile=opds-catalog;kind=acquisition,application/atom+xml;q=0.9,application/xml;q=0.8,*/*;q=0.1'`

Removing the `_get_accept_header` method, I got this accept header:
`'Accept': '*/*' `

It will probably break many things that are working. I think is better not touching it right now. What you think, @leonardr ?

Reply:
I don't see how you would end up with the `*/*` Accept header unless it was stored in your database configuration. But the branch works as it is so I agree it's not necessary to remove that code right now.